index.html

```diff
@@ -1,16 +1,90 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css" />
+    <script src="scripts.js"></script>
+  </head>
 
-<body>
+  <body>
     <!-- Add your markup here -->
-</body>
+    <header class="header" style="height: 32vh">
+      <div class="cake_shop_avatar">
+        <!-- <img src="img/cupcake.webp" /> -->
+      </div>
+      <div class="cake_shop_slogan">
+        <h1>The best cakes in town delivered to your door</h1>
+      </div>
+
+      <nav class="navbar">
+        <ul class="nav_menu">
+          <li class="nav_item"><a class="nav_link" href="#">Home</a></li>
+          <li class="nav_item"><a class="nav_link" href="#">Cakes</a></li>
+          <li class="nav_item"><a class="nav_link" href="#"> Ordering</a></li>
+          <li class="nav_item"><a class="nav_link" href="#">Lessons</a></li>
+          <li class="nav_item"><a class="nav_link" href="#">About</a></li>
+        </ul>
```

Comment:
At the moment, these links does not seems to do anything, is it still work in progress?